plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/AggregateMediatorPropertiesEditionPartForm.java

```diff
@@ -1518,34 +1521,42 @@ protected Composite createAggregationExpressionWidget(FormToolkit widgetFactory,
         widgetFactory.paintBordersFor(parent);
         GridData valueData = new GridData(GridData.FILL_HORIZONTAL);
         aggregationExpressionText.setLayoutData(valueData);
-        aggregationExpressionText.addFocusListener(new FocusAdapter() {
-            /**
-             * @see org.eclipse.swt.events.FocusAdapter#focusLost(org.eclipse.swt.events.FocusEvent)
-             * 
-             */
+        
+        aggregationExpressionText.addMouseListener(new MouseAdapter() {
+            
             @Override
-            @SuppressWarnings("synthetic-access")
-            public void focusLost(FocusEvent e) {
+            public void mouseDown( MouseEvent event ) {
+                openNamespacedPropertyEditor(parent);
             }
-
-            /**
-             * @see org.eclipse.swt.events.FocusAdapter#focusGained(org.eclipse.swt.events.FocusEvent)
-             */
+            
+        });
+        
+        aggregationExpressionText.addKeyListener(new KeyListener() {
+                        
             @Override
-            public void focusGained(FocusEvent e) {
-                EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(), SWT.NULL, aggregationExpression);
-                //valueExpression.setPropertyValue(valueExpressionText.getText());
-                nspd.open();
-                aggregationExpressionText.setText(aggregationExpression.getPropertyValue());
-                propertiesEditionComponent.firePropertiesChanged(new PropertiesEditionEvent(AggregateMediatorPropertiesEditionPartForm.this, EsbViewsRepository.AggregateMediator.OnComplete.aggregationExpression, PropertiesEditionEvent.COMMIT, PropertiesEditionEvent.SET, null, getAggregationExpression()));
+            public void keyPressed(KeyEvent e) {
+                openNamespacedPropertyEditor(parent);
             }
-        });
+            
+            @Override
+            public void keyReleased(KeyEvent e) {}
+            
+        });;
+        
         EditingUtils.setID(aggregationExpressionText, EsbViewsRepository.AggregateMediator.OnComplete.aggregationExpression);
         EditingUtils.setEEFtype(aggregationExpressionText, "eef::Text");
         Control aggregationExpressionHelp =FormUtils.createHelpButton(widgetFactory, parent, propertiesEditionComponent.getHelpContent(EsbViewsRepository.AggregateMediator.OnComplete.aggregationExpression, EsbViewsRepository.FORM_KIND), null); //$NON-NLS-1$
         aggregationExpressionElements = new Control[] {aggregationExpressionLabel, aggregationExpressionText, aggregationExpressionHelp};
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(), SWT.NULL, aggregationExpression);
+        //valueExpression.setPropertyValue(valueExpressionText.getText());
```

Comment:
Remove the commented line.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/800

Reply:
Fixed in https://github.com/dilee/devstudio-tooling-esb/commit/54fc62f60a4c5fefc16a70e7071a097b6487c6bf